README.md

```diff
@@ -111,3 +111,53 @@ add_action( 'experimental_features_flag_enabled_{feature-flag}', function() { ..
 ```php
 add_action( 'experimental_features_flag_disabled_{feature-flag}', function() { ... } );
 ```
+
+### Retrieving Feature Flag Status in the REST API.
+
+The status of feature flags can be retrieved via the REST API. The endpoint
+`/wp-json/experimental-features/v1/features` will return a JSON object with the
+status of all feature flags on the site.
+
+
+```json
+{
+	"my-cool-feature": {
+		"label": "My Cool Feature",
+		"status": false
+	}
+}
+```
+
+By default, this is disabled. To enable it, use the following filter:
```

Comment:
:100: Really like that this is opt-in. Experimental features could leak privileged information and I appreciate that care was taken to protect that!